t/data/empty_test_distribution/main.pm

```diff
@@ -0,0 +1,12 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use strict;
+use warnings;
+
```

Comment:
```suggestion
use Mojo::Base -strict;
```

I guess you wanted to test me :)

Also see all test modules in this repo like
https://github.com/os-autoinst/os-autoinst/blob/master/t/data/tests/bar/module2.pm